internal/generator/vector/input/viaq.go

```diff
@@ -124,7 +126,14 @@ func addLogType(spec logging.InputSpec, els []framework.Element, ids []string) (
 	}
 
 	if logType != "" {
-		id := helpers.MakeInputID(spec.Name, "viaq", "logtype")
+		compId := spec.Name
```

Comment:
Why did we need to make this change?  I don't see how the ID is no longer unique which is the primary issue

Reply:
This change was made because of an issue with input services. Input service names were not unique and would clash with multiple forwarders which would cause an endless reconcile loop if 2 forwarders had the same http input name.

Reply:
This is an issue across multiple CLF and needs to be addressed during:
* service creation
* maybe any secrets/configmaps we create

The actual config has a 1:1 relationship with the CLF and is only visible to the collector.  Am I missing why we need to propagate those changes to the config?

Reply:
I see what you mean and I think you are correct here. I will make the appropriate changes.